### PR TITLE
Allow multi-page content query

### PR DESCRIPTION
**Pull request by hfern**
I guess it introduces ErrPageNotFound as well.

### CODE REVIEW

**Comment by hfern** [comment]
Also I release the patches submitted under the Creative Commons CC0 public domain dedication.

File: edit.go [hunk 1]
```diff
@@ -88,43 +107,98 @@ func (w *Client) getPage(pageIDorName string, isName bool) (content string, time
...
+			p.Add("titles", identifier)
+		} else {
+			p.Add("pageids", identifier)
+		}
 	}
 
 	resp, err := w.Get(p)
 	if err != nil {
-		return "", "", err
+		return pages, err
```
Comment (cgt) [inline edit.go]: Why return the empty `pages` map instead of `nil` here (and other similar places in this method)?

**Comment by cgt** [comment]
Superficially, it looks reasonable. I will have a closer look tonight and merge it soon. Thanks for the contribution.

File: edit.go (hunk 1 above)
Comment (hfern) [inline edit.go]: You're definitely right. API contract errors should return a nil map.

**Pull request merged**